### DISCUSSION TITLE
ControllerLauncher in version 2023.0.4

### DISCUSSION THREAD

**Discussion by Sciroccogti** [cyberbotics/webots_ros2#844] [Q&A]
We used to use
```python
from webots_ros2_core.utils import ControllerLauncher
```

Now that `webots_ros2_core` is deprecated since 2023.0.0, which can be used as alternative?

We are on foxy, Ubuntu22.04.

Thanks!

**Comment by omichel** [comment]
You should use `webots_ros2_driver` instead.
See the [basic tutorial](https://docs.ros.org/en/iron/Tutorials/Advanced/Simulators/Webots/Setting-Up-Simulation-Webots-Basic.html) and [getting started documentation](https://github.com/cyberbotics/webots_ros2/wiki/Getting-Started).

**Reply by Sciroccogti** [reply]
I noticed that most of `webots_ros2_core` has been merged into `webots_ros2_driver`, but not `ControllerLauncher`. Is it moved into other package, or just removed?

**Reply by omichel** [reply]
It is replaced by [WebotsController](https://github.com/cyberbotics/webots_ros2/wiki/References-Nodes) in `webots_ros2_driver`.
